chart/README.md

```diff
@@ -540,6 +540,29 @@ Some example commands are shown below.
    .
    ```
 
+#### Installing from remote repository
+```
+helm repo add sonatype https://sonatype.github.io/helm3-charts/
+
+helm install --namespace staging mycluster --dependency-update \
```

Comment:
With this I think we should probably get rid of the other examples that we installing from the repo clone dir. (or update them to us this approach)

Reply:
I updated the existing examples replacing sensitive data.